apps/web/src/server/lib/nunjucks-filters/index.js

```diff
@@ -30,7 +30,6 @@ export { userTypeMap } from './user-type-map.js';
 export { MIME } from './mime-type.js';
 export { fileType } from './mime-type.js';
 export { statusName } from './status-name.js';
-export { documentName } from './document-name.js';
```

Comment:
this is not necessary anymore: the backend does the same thing (it was the script that removes the extension from the file name)